src/client/components/HealthCareApp.jsx

```diff
@@ -173,19 +179,36 @@ class HealthCareApp extends React.Component {
         </div>
       );
     }
+    let devPanel = undefined;
+    if (__DEV__) {
+      const queryParams = _.fromPairs(
+        window.location.search.substring(1).split('&').map((v) => { return v.split('='); }));
```

Comment:
Nice!  I was looking for something like that but couldn't find it.  I'll fix in a follow-up PR.